src/assets/styles/scroll.less

```diff
@@ -0,0 +1,40 @@
+.searchScroll {
+  top: 52px;
+  bottom: 0px;
+  position: fixed;
+  padding-bottom: 30px;
+  padding-bottom: 0px;
+}
+
+.searchScroll .mCSB_scrollTools {
+  left: 2px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.favScroll {
+  top: 52px;
+  bottom: 0px;
```

Comment:
at the moment they are identical, we need 2 classes to target them with javascript separately. could add 3rd class for styling. or have class for styling target with ids